src/main/java/com/experoinc/janusgraph/diskstorage/foundationdb/FoundationDBTx.java

```diff
@@ -197,6 +196,15 @@ private TransactionClose(String msg) {
         return result;
     }
 
+    private <T> T getTransaction(IsolationLevel isolationLevel, Transaction tx) {
+        if(IsolationLevel.READ_COMMITTED_NO_WRITE.equals(isolationLevel)
```

Comment:
https://apple.github.io/foundationdb/developer-guide.html#snapshot-reads
With snapshot reads, reads will see last successfully committed transaction version of the key. The key values may change after the read occurred.